SCClassLibrary/Common/Control/Node.sc

```diff
@@ -463,12 +469,10 @@ Synth : Node {
 	}
 
 	newMsg { arg target, args, addAction = \addToHead;
-		var addNum, inTarget;
-		addNum = addActions[addAction];
-		// if target is nil set to default group of server specified when basicNew was called
```

Comment:
maybe better to leave this comment - it's not immediately clear that this is what happens when you call asTarget on nil (as I just showed :P)

Reply:
In all other cases, there are no comments. This behavior is just the standard for `asTarget` (it's documented in the "asTarget" helpfile).

Reply:
ok! line 263 above still has it. maybe it should be removed there too for consistency then